components/alert/style/index.less

```diff
@@ -132,9 +133,8 @@
   &&-motion-leave {
     overflow: hidden;
     opacity: 1;
-    transition: max-height 0.3s @ease-in-out-circ, opacity 0.3s @ease-in-out-circ,
-      padding-top 0.3s @ease-in-out-circ, padding-bottom 0.3s @ease-in-out-circ,
-      margin-bottom 0.3s @ease-in-out-circ;
+    transition: max-height 0.3s @ease-in-out-circ, opacity 0.3s @ease-in-out-circ, padding-top 0.3s @ease-in-out-circ,
+      padding-bottom 0.3s @ease-in-out-circ, margin-bottom 0.3s @ease-in-out-circ;
```

Comment:
Move custom styles to `patch.less`, we should not change the file `index.less` which is synced from `ant-design`